geonode_mapstore_client/client/js/components/Tabs/Tabs.jsx

```diff
@@ -16,11 +16,12 @@ const Tabs = ({
     identifier,
     selectedTabId,
     onSelect,
+    activeKey,
     className
 }) => {
     const [eventKeys, setEventKeys] = useLocalStorage('tabSelected', {});
     const persistSelection = isNil(selectedTabId);
-    const selectedKey = !persistSelection ? selectedTabId : (eventKeys[identifier] ?? 0);
+    const selectedKey = activeKey || (!persistSelection ? selectedTabId : (eventKeys[identifier] ?? 0));
```

Comment:
We should use selectedTabId instead

Reply:
Probably, I thought this should avoid persist when we simply was to use activeKey and manage it with onSelect